### PR TITLE
`Development`: Add various e2e fixes and enhancements

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Description

This PR fixes the following things:
- It renames `PostgreSQL` to `Postgres` for the sorry cypress setup
- It updates the cypress docker image to the latest version
- It sets a upload limit for the minio bucket (overwrites the default 1MB limit)
- It sets the cypress video upload, to only upload on failure (this will avoid unnecessary files in the Minio storage)
- It sets the `NO_COLOR` option of cypress, so the bitbucket raw log output becomes more readable
- It removes `cypress` from the test data 
- It extends the UUID generation from 5 chars to 9 chars to avoid name clashes
- It updates `cypress-cloud` to `1.9.0`

#### Code Review
- [x] Review 1
- [ ] Review 2

### CODE REVIEW

**Review by Hialus — APPROVED** [review]
Changes look good

File: src/test/cypress/support/requests/CourseManagementRequests.ts [hunk 1]
```diff
@@ -133,7 +133,7 @@ export class CourseManagementRequests {
         releaseDate = day(),
         dueDate = day().add(1, 'day'),
         title = 'Programming ' + generateUUID(),
-        programmingShortName = 'cypress' + generateUUID(),
+        programmingShortName = 'programming' + generateUUID(),
```
Comment (4ludwig4) [inline src/test/cypress/support/requests/CourseManagementRequests.ts:136]: Just out of curiosity, why did you change from the cypress prefix to the programming and other prefixes, etc. ? I just know that in the pre-live Jira it might be helpful to see what's coming from cypress?!

File: src/test/cypress/support/utils.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ day.extend(utc);
  * */
 export function generateUUID() {
     const uuid = uuidv4().replace(/-/g, '');
-    return uuid.substr(0, 5);
+    return uuid.substr(0, 9);
```
Comment (4ludwig4) [inline src/test/cypress/support/utils.ts:18]: Are the UUIDs used anywhere else in some regexes etc? Just asking as this might be a problem then but guess not.

**Review by 4ludwig4 — APPROVED** [review]
The code looks good from my side.
Just some comments which shouldn't be any issues.
Doublechecked postgres refactoring and couldn't find any related cypress changes.

File: src/test/cypress/support/utils.ts (hunk 2 above)
Comment (TheZoker) [reply src/test/cypress/support/utils.ts:18]: No they are only used for unique entities within artemis

File: src/test/cypress/support/requests/CourseManagementRequests.ts (hunk 1 above)
Comment (TheZoker) [reply src/test/cypress/support/requests/CourseManagementRequests.ts:136]: We want to stay consistent across the E2E test. Since they are deleted anyway, this is not an issue
